app/src/debug/res/raw/initial_experiments.json

```diff
@@ -4,46 +4,53 @@
           "appId": "org.mozilla.fenix",
           "appName": "fenix",
           "channel": "nightly",
-          "branches": [{
-                  "slug": "control",
-                  "ratio": 100,
-                  "feature": {
-                      "value": {},
-                      "enabled": true,
-                      "featureId": "nimbus-validation"
-                  }
-              },
+          "branches": [
               {
-                  "slug": "fancy-settings",
+                  "slug": "no-mr2",
                   "ratio": 0,
                   "feature": {
                       "value": {
-                          "settings-title": "Fancy Settings"
+                           "sections-enabled": {
+                               "recentExplorations": true,
+                               "recentlySaved": false,
+                               "jumpBackIn": false,
+                               "pocket": false
+                          }
                       },
                       "enabled": true,
-                      "featureId": "nimbus-validation"
+                      "featureId": "homescreen"
                   }
               },
               {
-                  "slug": "smiley",
+                  "slug": "full-mr2",
                   "ratio": 0,
                   "feature": {
                       "value": {
-                          "settings-title-punctuation": "\uD83D\uDE03"
+                          "sections-enabled": {
+                              "recentExplorations": true,
```

Comment:
I renamed this section to be `recentExplorations` as we are not going to put top sites under experimentation